solutions/CF Mixing Water.mdx

```diff
@@ -36,41 +38,45 @@ using namespace std;
 double hot_water, cold_water, desired_temp;
 
 /*
-	This function checks whether pouring cupsPoured amount of cups achieves an average
-	barrel temperature greater than or equal to desired_temp. Technically, this represents
-	2 * cupsPoured + 1 cups
+	* This function checks whether pouring cupsPoured amount of cups achieves an average
+	* barrel temperature greater than or equal to desired_temp. Technically, this represents
+	* 2 * cupsPoured + 1 cups
 */
 
 bool works(int cupsPoured){
 
 	double totalBarrelTemperature = (hot_water * (cupsPoured + 1) + cold_water * (cupsPoured));
 	double totalCupsPoured = (double) (2 * cupsPoured + 1.0);
 	double avgBarrelTemperature = totalBarrelTemperature / totalCupsPoured;
-
 	return avgBarrelTemperature >= desired_temp;
 
 }
 
 void solve(){
 
-	/* maximumCups represents the the maximum number of cups such
-	that the barrel temperature is greater than desired_temp.
+	/* 
+	   * maximumCups represents the the maximum number of cups such
+	   * that the barrel temperature is greater than desired_temp.
 	*/
-
 	cin >> hot_water >> cold_water >> desired_temp;
-
 	// Binary search on the answer for maximumCups.
-
 	int left = 0, right = 1e6 + 1, maximumCups = 0;
 
 	while(left <= right){
+
 		int mid = left + (right - left) / 2;
+
 		if(works(mid)){
+		
 			maximumCups = mid;
 			left = mid + 1;
+			
 		} else {
+		
 			right = mid - 1;
+			
 		}
+
```

Comment:
are all these extra lines intended lmao

Reply:
I think you misinterpreted; add white space between parentheses and curly brackets, not the lines lol